meteor/client/lib/notifications/notifications.ts

```diff
@@ -477,13 +477,20 @@ export class Notification extends EventEmitter {
 }
 
 export function getNoticeLevelForPieceStatus(statusCode: RundownAPI.PieceStatusCode): NoticeLevel | null {
-	return statusCode !== RundownAPI.PieceStatusCode.OK && statusCode !== RundownAPI.PieceStatusCode.UNKNOWN
-		? statusCode === RundownAPI.PieceStatusCode.SOURCE_NOT_SET
-			? NoticeLevel.CRITICAL
-			: // : innerPiece.status === RundownAPI.PieceStatusCode.SOURCE_MISSING ||
-			  // innerPiece.status === RundownAPI.PieceStatusCode.SOURCE_BROKEN
-			  NoticeLevel.WARNING
-		: null
+	if (statusCode === RundownAPI.PieceStatusCode.OK || statusCode === RundownAPI.PieceStatusCode.UNKNOWN) {
+		return null
```

Comment:
Returning null is generally a bad practice, but I see that the code allready did that before the refactor.

Reply:
yes, I changed as little as possible with the refactor.